Ruddarr/Ruddarr.swift

```diff
@@ -12,8 +13,12 @@ struct Ruddarr: App {
     init() {
         #if DEBUG
         // dependencies = .mock
+        // Tips.showAllTipsForTesting()
         #endif
 
+        try? Tips.configure()
+        // try? Tips.resetDatastore()
+
```

Comment:
I like the `showAllTipsForTesting` here, but we can't configure the tips globally right now, because we loose the `series.monitored` condition. I don't think Tip.rules are sophisticated enough for us to use.

Reply:
Maybe it's good to do it only once, but we'll need to handle the `series.monitored` condition somehow.